wix/wrapper/wrapper_windows.go

```diff
@@ -45,6 +45,10 @@ func main() {
 		return
 	}
 
+	err := eventlog.InstallAsEventCreate(name, eventlog.Error|eventlog.Warning|eventlog.Info)
+	if err != nil {
+		log.Fatal("SetupEventLogSource() failed: %s", err)
```

Comment:
```
wix\wrapper\wrapper_windows.go:50: possible formatting directive in Fatal call
```

You should use `Fatalf` instead. ref: https://github.com/stretchr/testify/issues/339